.github/workflows/spec.yaml

```diff
@@ -12,15 +12,15 @@ jobs:
       with:
         bundler-cache: true
     - run: bundle exec rspec
-    - uses: docker/build-push-action@v1
+    - uses: docker/build-push-action@master
```

Comment:
I don't think we use the Docker image for anything anymore, so it might be worth removing this entirely, but perhaps we can do it separately.